roles/openshift_aws/defaults/main.yml

```diff
@@ -244,6 +244,11 @@ openshift_aws_node_security_groups:
       from_port: all
       to_port: all
       group_name: "{{ openshift_aws_clusterid }}"
+    - proto: tcp
```

Comment:
I think we should default this for GCP as well.  openshift_node_open_ports is supposed to get added to the firewall rules.  If we default openshift_node_open_ports, then loop through it to set this default, we can keep it consistent between both

@kwoodson

Reply:
The default on GCP is:

```
openshift_node_open_ports: [{"service":"Router stats port", "port":"1936/tcp"}, {"service":"Allowed open host ports", "port":"9000-10000/tcp"}, {"service":"Allowed open host ports", "port":"9000-10000/udp"}]
```

Reply:
@smarterclayton, Are you wanting to combine these ports with the currently defined loadbalancer ports?

Reply:
The line above this allows:
```
    - proto: all
      from_port: all
      to_port: all
      group_name: "{{ openshift_aws_clusterid }}"
```

Is this change necessary?  If we remove the all to all then this should be good but with how it stands today it should work.  Maybe I missed something.

Reply:
I confirmed this change is necessary.